### DISCUSSION TITLE
Change -Ofast flag for clang and clang++ into -O3 -ffast-math

### DISCUSSION THREAD

**Discussion by R-Goc** [PlummersSoftwareLLC/Primes#976] [General]
Starting with LLVM 19 building the c++ examples containing a -Ofast flag will raise the following warning: 
```
clang++: warning: argument '-Ofast' is deprecated; use '-O3 -ffast-math' for the same behavior, or '-O3' to enable only conforming optimizations [-Wdeprecated-ofast]
```
I can make a PR for the changes on clang. Flang will also have this change I believe, not sure if it is used in this project.
RFC for this:
[https://discourse.llvm.org/t/rfc-deprecate-ofast/78687](url)

**Comment by rbergen** [comment]
@R-Goc If you open a PR for this then I'm happy to review it. Please do consider that the base images used for the various solutions that employ clang tend to include clang versions that are well before 1.19, and whatever changes your PR includes can't break the solutions in question - or degrade their performance.

The Fortran solutions within the Primes project seem to be compiled using gfortran.
